doc/man3/SSL_set1_server_cert_type.pod

```diff
@@ -0,0 +1,85 @@
+=pod
+
+=head1 NAME
+
+SSL_set1_client_cert_type,
+SSL_set1_server_cert_type,
+SSL_CTX_set1_client_cert_type,
+SSL_CTX_set1_server_cert_type,
+SSL_get0_client_cert_type,
+SSL_get0_server_cert_type,
+SSL_CTX_get0_client_cert_type,
+SSL_CTX_get0_server_cert_type - certificate type (RFC7250) support
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_set1_client_cert_type(SSL *s, const unsigned char *val, size_t len);
+ int SSL_set1_server_cert_type(SSL *s, const unsigned char *val, size_t len);
+ int SSL_CTX_set1_client_cert_type(SSL_CTX *ctx, const unsigned char *val, size_t len);
+ int SSL_CTX_set1_server_cert_type(SSL_CTX *ctx, const unsigned char *val, size_t len);
+ int SSL_get0_client_cert_type(SSL *s, unsigned char **t, size_t *len);
+ int SSL_get0_server_cert_type(SSL *s, unsigned char **t, size_t *len);
+ int SSL_CTX_get0_client_cert_type(SSL_CTX *ctx, unsigned char **t, size_t *len);
+ int SSL_CTX_get0_server_cert_type(SSL_CTX *s, unsigned char **t, size_t *len);
+
+=head1 DESCRIPTION
+
+The SSL_set1_client_cert_type() and SSL_CTX_set1_client_cert_type() set the
+values for the client certificate type extension. The SSL_get0_client_cert_type()
+and SSL_CTX_get0_client_cert_type() retreive the values to be used in the client
+certificate type extension.
```

Comment:
It is likely not obvious to the uninitiated that the "client certificate type" (plural?) are the types that are acceptable to the server to be sent **by** the client, rather than the ones that the client signals to the server as acceptable **from** the server.  The API docs should make the intended producer/consumer more clear.